_emails/erie_pa.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Erie, PA
+permalink: "/erie_pa"
```

Comment:
I would just use /erie here, no need to include state as well

Reply:
Done.

Reply:
Also changed the file name from `erie_pa.md` to `erie.md`